cmd/daytona/main.go

```diff
@@ -211,6 +212,21 @@ func main() {
 	// users to override this default by using the VAULT_MAX_RETRIES env var
 	vaultConfig.MaxRetries = 5
 
+	
+	// Set the MaxRetryWait for rate-limited requests to a higher default, but allow
+	// users to override this default by using the VAULT_MAX_RETRY_WAIT env var. If we 
+	// minute-length rate limiting buckets with some jitter, the Retry-After header could
+	// indicate upto 69 seconds of wait time, so we want Vault to respect that.
+	vaultConfig.SetMaxRetryWait(70 * time.Second)
```

Comment:
Should this be set in `api.DefaultConfig()`?

Reply:
That's Hashicorp code right?

I'll probably open a PR for them to add this config knob upstream. We're still not 100% sure how all the knobs work together in the Vault client code so this PR is sort of a placeholder for us until we're certain how it all goes together from some testing.

Reply:
oh. . . for some reason I was thinking that was our code.